addons/website_form/static/src/snippets/s_website_form/options.js

```diff
@@ -1222,6 +1247,53 @@ options.registry.AddField = FieldEditor.extend({
     },
 });
 
+options.registry.WebsiteFormSubmit = options.Class.extend({
+    /**
+     * Select the submit button's position
+     */
+    selectButtonPosition: function (previewMode, value, params) {
+        this.$target[0].querySelectorAll('.s_website_form_label').forEach(el => el.remove());
+        this.$target[0].classList.remove('text-center');
+        const button = this.$target[0].querySelector('.s_website_form_send');
+        button.classList.remove('float-right', 'float-left');
+        switch (value) {
+            case 'center':
+                this.$target[0].classList.add('text-center');
+                break;
+            case 'align': {
+                const div = document.createElement('div');
+                const formOptions = this.el.closest('.o_we_customize_panel');
+                div.style.width = formOptions && formOptions.querySelector('we-customizeblock-options [data-select-style] input').value + 'px',
+                div.classList = 's_website_form_label';
+                this.$target[0].insertBefore(div, button);
+                break;
```

Comment:
I wonder why you are only dealing with this div when we choose align... it should be dealt with every time I think?

Reply:
Well I remove it every time and only add it when align is selected as it is useless in the other cases.

Reply:
Ok, I did not see you remove it every time, my bad then